### PR TITLE
Bump babel-loader to 7.1.1

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.15
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
I *think* this is required to help fix Foreman breakages.

File: nodejs-babel-loader/nodejs-babel-loader.spec [hunk 1]
```diff
@@ -1,95 +1,92 @@
...
 
 %files
 %{nodejs_sitelib}/%{npm_name}
 
-%files doc
+%doc CHANGELOG.md
 %doc README.md
 
 %changelog
-* Sat Sep 24 2016 Eric D Helms <ericdhelms@gmail.com> 6.2.4-2
```
Comment (mmoll) [inline nodejs-babel-loader/nodejs-babel-loader.spec]: why remove this?

**Comment by mmoll** [comment]
nodejs-babel-loader/babel-loader-7.1.1-registry.npmjs.org.tgz is not in annex

**Comment by ehelms** [comment]
Updated

**Comment by ehelms** [comment]
@mmoll It appears to be normal for that file to be checked in as a binary. That's how it was before this update.

**Comment by mmoll** [comment]
now `babel-loader-7.1.1.tgz` and `babel-loader-7.1.1-registry.npmjs.org.tgz` is there, but the annex get is failing for `babel-loader-7.1.1-registry.npmjs.org.tgz`

**Comment by ehelms** [comment]
@mmoll now we are passing

**Comment by mmoll** [comment]
merge, thanks @ehelms!

**Pull request merged**